qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/jupyterhub/files/jupyterhub/01-theme.py

```diff
@@ -1,10 +1,9 @@
+import z2jh
```

Comment:
These files should NOT be modified -- the import order is important here, this might fix the Kubernetes tests, @iameskild add an exception to the python linting to avoid this dir.

Reply:
Awesome!! Thanks a lot for this find @viniciusdc! I'll go ahead and make these changes. Do you know why this ordering matters?

Reply:
I do know exactly the problem, as I only encountered it when importing the KubeSpwaner. But it seems that by changing the order some extra configurations are not imported correctly and because of this, some settings ends up with missing imports... something on these lines. I will try to find a more elaborate answer.

Reply:
Thanks @viniciusdc! I was just curious since I haven't come across an issue like this before.